apps/src/code-studio/assets/DefaultSpriteRow.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+
+export default class DefaultSpriteRow extends React.Component {
```

Comment:
gah, i missed this in my prior review, but we should make this a functional component since it's stateless (react 16.8 allows all components to be functional, and eventually class components will be deprecated, which is mostly a concern for third-party dependencies and being able to upgrade them).

[StylizedBaseDialog.jsx](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/componentLibrary/StylizedBaseDialog.jsx) has two functional components, but the basic structure will look like this:

```javascript
export default function DefaultSpriteRow(props) {
  // no render function
  return <div>{props.name}</div>;
}

DefaultSpriteRow.propTypes = {
  name: PropTypes.string.isRequired
};


```